packages/component/src/Middleware/AttachmentForScreenReader/createCoreMiddleware.js

```diff
@@ -17,7 +17,7 @@ export default function createCoreMiddleware() {
         }
       ] = args;
 
-      return role === 'user' && !/^text\//u.test(contentType) && !thumbnailUrl
+      return role === 'user' && !/^audio|image|video\//u.test(contentType) && !attachment.content && !thumbnailUrl
```

Comment:
why are checking if the  attachment.content does not exist? is this the expected behavior with text  file Content?

Reply:
If the attachment doesn't exist, the text renderer is used to render the text of an activity. This is the original cause of the error.